src/client/app/common/views/components/settings/2fa.vue

```diff
@@ -1,11 +1,55 @@
 <template>
-<div class="2fa">
+<div class="2fa totp-section">
 	<p style="margin-top:0;">{{ $t('intro') }}<a :href="$t('url')" target="_blank">{{ $t('detail') }}</a></p>
 	<ui-info warn>{{ $t('caution') }}</ui-info>
 	<p v-if="!data && !$store.state.i.twoFactorEnabled"><ui-button @click="register">{{ $t('register') }}</ui-button></p>
 	<template v-if="$store.state.i.twoFactorEnabled">
+		<h2 class="heading">{{ $t('totp-header') }}</h2>
 		<p>{{ $t('already-registered') }}</p>
 		<ui-button @click="unregister">{{ $t('unregister') }}</ui-button>
+
+
+		<hr v-if="supportsCredentials" class="totp-method-sep">
+
+		<template v-if="supportsCredentials">
```

Comment:
Is it possible to move the above `<hr>` into `<template>`?

Reply:
@rinsuki Yes, but I moved it outside because it separates the two different types of credentials which was why I thought it made more sense outside